docs/source/conf.py

```diff
@@ -108,6 +108,9 @@
 # A list of ignored prefixes for module index sorting.
 #modindex_common_prefix = []
 
+# List of Sphinx warnings that will not be raised
+suppress_warnings = ['epub.unknown_project_files']
```

Comment:
Unless I put this here, build fails due to `sphinx.errors.SphinxWarning: unknown mimetype for _static/zeromq.ico, ignoring` because `.ico` extension is not defined in [sphinx.builders._epub_base](https://github.com/sphinx-doc/sphinx/blob/v3.0.4/sphinx/builders/_epub_base.py#L67).

<details>
<summary>Complete stack trace</summary>

```
Running Sphinx v3.0.4
loading translations [en]... done
making output directory... done
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from http://www.tornadoweb.org/en/stable/objects.inv...
intersphinx inventory has moved: http://www.tornadoweb.org/en/stable/objects.inv -> https://www.tornadoweb.org/en/stable/objects.inv
building [mo]: targets for 0 po files that are out of date
building [epub]: targets for 29 source files that are out of date
updating environment: [new config] 29 added, 0 changed, 0 removed
reading sources... [  3%] api/index
reading sources... [  6%] api/zmq
reading sources... [ 10%] api/zmq.asyncio
reading sources... [ 13%] api/zmq.auth
reading sources... [ 17%] api/zmq.auth.ioloop
reading sources... [ 20%] api/zmq.auth.thread
reading sources... [ 24%] api/zmq.decorators
reading sources... [ 27%] api/zmq.devices
reading sources... [ 31%] api/zmq.eventloop.future
reading sources... [ 34%] api/zmq.eventloop.ioloop
reading sources... [ 37%] api/zmq.eventloop.zmqstream
reading sources... [ 41%] api/zmq.green
reading sources... [ 44%] api/zmq.log.handlers
reading sources... [ 48%] api/zmq.ssh.tunnel
reading sources... [ 51%] api/zmq.utils.jsonapi
reading sources... [ 55%] api/zmq.utils.monitor
reading sources... [ 58%] api/zmq.utils.win32
reading sources... [ 62%] api/zmq.utils.z85
reading sources... [ 65%] changelog
reading sources... [ 68%] devices
reading sources... [ 72%] draft
reading sources... [ 75%] eventloop
reading sources... [ 79%] index
reading sources... [ 82%] logging
reading sources... [ 86%] morethanbindings
reading sources... [ 89%] pyversions
reading sources... [ 93%] serialization
reading sources... [ 96%] ssh
reading sources... [100%] unicode

looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [  3%] api/index
writing output... [  6%] api/zmq
writing output... [ 10%] api/zmq.asyncio
writing output... [ 13%] api/zmq.auth
writing output... [ 17%] api/zmq.auth.ioloop
writing output... [ 20%] api/zmq.auth.thread
writing output... [ 24%] api/zmq.decorators
writing output... [ 27%] api/zmq.devices
writing output... [ 31%] api/zmq.eventloop.future
writing output... [ 34%] api/zmq.eventloop.ioloop
writing output... [ 37%] api/zmq.eventloop.zmqstream
writing output... [ 41%] api/zmq.green
writing output... [ 44%] api/zmq.log.handlers
writing output... [ 48%] api/zmq.ssh.tunnel
writing output... [ 51%] api/zmq.utils.jsonapi
writing output... [ 55%] api/zmq.utils.monitor
writing output... [ 58%] api/zmq.utils.win32
writing output... [ 62%] api/zmq.utils.z85
writing output... [ 65%] changelog
writing output... [ 68%] devices
writing output... [ 72%] draft
writing output... [ 75%] eventloop
writing output... [ 79%] index
writing output... [ 82%] logging
writing output... [ 86%] morethanbindings
writing output... [ 89%] pyversions
writing output... [ 93%] serialization
writing output... [ 96%] ssh
writing output... [100%] unicode

generating indices...  genindex py-modindexdone
writing additional pages... done
copying static files... ... done
copying extra files... done
writing mimetype file...
writing META-INF/container.xml file...
writing content.opf file...

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app.build(args.force_all, filenames)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/application.py", line 348, in build
    self.builder.build_update()
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 299, in build_update
    len(to_build))
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 364, in build
    self.finish()
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/builders/html/__init__.py", line 622, in finish
    self.handle_finish()
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/builders/epub3.py", line 79, in handle_finish
    self.build_content()
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/builders/_epub_base.py", line 541, in build_content
    type='epub', subtype='unknown_project_files')
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 1736, in warning
    self.log(WARNING, msg, *args, **kwargs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/util/logging.py", line 125, in log
    super().log(level, msg, *args, **kwargs)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 1768, in log
    self.logger.log(level, msg, *args, **kwargs)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 1449, in log
    self._log(level, msg, args, **kwargs)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 1519, in _log
    self.handle(record)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 1529, in handle
    self.callHandlers(record)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 1591, in callHandlers
    hdlr.handle(record)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 901, in handle
    rv = self.filter(record)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/logging/__init__.py", line 762, in filter
    result = f.filter(record)
  File "/home/docs/checkouts/readthedocs.org/user_builds/sakurai-youhei-pyzmq/envs/latest/lib/python3.7/site-packages/sphinx/util/logging.py", line 417, in filter
    raise SphinxWarning(message)
sphinx.errors.SphinxWarning: unknown mimetype for _static/zeromq.ico, ignoring

Warning, treated as error:
unknown mimetype for _static/zeromq.ico, ignoring
```
</details>